### ISSUE TITLE
Witch's familiar crashes game in co-op combat

### ISSUE THREAD

**Issue by gobsmacked1** [SolastaMods/SolastaCommunityExpansion#1554]
We were playing a 6-character, 2-player network game.  Witch summoned her familiar.  Everything was fine until we entered combat, and the moment the combat ticker reached the owl's turn in initiative it crashed both the client and hoster.

We reloaded from a network save, and proceeded to try to get rid of the owl.  But since the owl is a permanent familiar, and there is no un-summon button, we were very lucky to have someone else in the party with Burning Hands to be able to fry the owl.

After the owl was out of the party, combat flowed without issue.  In hindsight, un-summoning might be a good free action to enable.  Pets can cause trouble.  Thanks.

[Network_Party_OwlFriend.zip](https://github.com/SolastaMods/SolastaCommunityExpansion/files/9125848/Network_Party_OwlFriend.zip)

**Comment by ThyWoof** [comment]
Share a log as well

**Comment by gobsmacked1** [comment]
No idea which logs you mean.  Hopefully one of these two files.  I don't have access to the client player's logs.  I was the host.  Thanks again.

[Solasta.zip](https://github.com/SolastaMods/SolastaCommunityExpansion/files/9126743/Solasta.zip)

**Comment by ThyWoof** [comment]
just to be clear. Are you playing a MP game with 6 heroes? Please confirm if indeed the case as this is unsupported and if so not sure how you're able to.

**Comment by gobsmacked1** [comment]
Correct, I created the game initially as a single player, 6-character SAV.  Then I loaded that SAV from the multiplayer menu, and had the other player choose three characters from my list.  We've been playing it for hours; not a single crash with the exception of the witch's owl familiar.  Haven't cast any pets since then as a precaution.  

I have since saved and reloaded multiplayer SAVs many times--no bugs.

I know it's currently not possible to create a six-character multiplayer game from the multiplayer menu.  But a few days ago I got the idea to try loading an existing single player game from the multiplayer menu, and voila.
